ocs_ci/ocs/ui/base_ui.py

```diff
@@ -743,7 +743,7 @@ def wait_for_namespace_selection(self, project_name):
 
         from ocs_ci.ocs.ui.helpers_ui import format_locator
 
-        if self.ocp_version_full >= version.VERSION_4_10:
+        if version.VERSION_4_10 <= self.ocp_version_full < version.VERSION_4_12:
```

Comment:
I think it's more readable
```
if self.ocp_version_full in (version.VERSION_4_10, version.VERSION_4_11):
```

Reply:
I agree, thanks. I assume it is not showstopper, I will fix it with next PR's I have 2 more BZ to cover over this vicinity